.github/workflows/test-notify-slack.yml

```diff
@@ -0,0 +1,40 @@
+---
+
+name: 'Notify slack'
+
+on:
+  push:
+  workflow_call:
+    secrets:
+      slack_webhook_url:
+        required: true
+    # inputs:
+    #   application:
+    #     description: 'The friendly name for the application'
+    #     type: string
+    #     required: true
+    #   environment:
+    #     description: 'The environment that is being targeted for this deployment'
+    #     type: string
+    #     required: true
+
+permissions:
+  id-token: write # aws auth
+  contents: write # publish release assets
```

Comment:
We do not need any permissions here.  The id-token is for aws auth which we are not using and we do not need to write anything to github itself.